hack/cluster-clean.sh

```diff
@@ -141,18 +149,43 @@ function delete_namespaces() {
 }
 
 function main() {
-    echo "Cleaning up ..."
+    SKIP_CLEAN="false"
+    SKIP_WAIT="false"
```

Comment:
- These should be local, otherwise these become global.
- Looks to me like bolleans are more appropriate here than string. (`SKIP_WAIT=false`)
When you compare, a simple `[ $SKIP_WAIT == false ]` will do the job.